src/Sylius/Behat/Context/Transform/TaxonContext.php

```diff
@@ -32,15 +32,14 @@ public function __construct(TaxonRepositoryInterface $taxonRepository)
     {
         $this->taxonRepository = $taxonRepository;
     }
-
+    
     /**
      * @Transform /^classified as "([^"]+)"$/
      * @Transform /^belongs to "([^"]+)"$/
-     * @Transform /^"([^"]+)" taxon$/
-     * @Transform /^"([^"]+)" as a parent taxon$/
-     * @Transform /^"([^"]+)" parent taxon$/
-     * @Transform /^parent taxon to "([^"]+)"$/
-     * @Transform /^taxon with "([^"]+)" name/
+     * @Transform /^taxon (?:|to |should be )"([^"]+)"$/
+     * @Transform /^"([^"]+)" (?:|as a parent |parent )taxon$/
+     * @Transform /^(?:a|an) "([^"]+)"$/
```

Comment:
I'm not sure about this transformer. I bet it could cause a weird bugs or blow up a whole app if another similar transformer would be placed somewhere else...